### PR TITLE
Quickfix demo-spell

### PR DESCRIPTION
**Pull request by nvasilakis**
Quick fix for #218

### CODE REVIEW

File: evaluation/intro/demo-spell.sh [hunk 1]
```diff
@@ -1,10 +1,14 @@
-#!/bin/bash
+#!/bin/sh
```
Comment (angelhof) [inline evaluation/intro/demo-spell.sh:1]: Why is this change necessary?

**Review by angelhof — APPROVED** [review]

File: evaluation/intro/demo-spell.sh (hunk 1 above)
Comment (nvasilakis) [reply evaluation/intro/demo-spell.sh:1]: It's not necessary, just want to make sure this example runs with `sh` (and it does)
